src/Microsoft.IdentityModel.TestExtensions/TestTokenCreator.cs

```diff
@@ -258,9 +258,9 @@ public string CreateTokenWithMissingIssuer()
             var tokenDescriptor = CreateTokenDescriptorWithInstanceOverrides();
 
             // Remove the default value and null out any class specific override.
-            tokenDescriptor.Subject.RemoveClaim(
+            tokenDescriptor?.Subject?.RemoveClaim(
```

Comment:
this doesn't make sense. we handle the null case then 2 lines down we state it is not null?

Can CreateTokenDescriptorWithInstanceOverrides return null?

Reply:
only needed here:

`tokenDescriptor.Subject?.RemoveClaim(`